main/pipeline/pipeline.cc

```diff
@@ -1014,7 +1001,9 @@ ast::ParsedFilesOrCancelled typecheck(unique_ptr<core::GlobalState> &gs, vector<
                                                 make_move_iterator(threadResult.trees.end()));
                     }
                     cfgInferProgress.reportProgress(fileq->doneEstimate());
-                    gs->errorQueue->flushErrors();
+                    for (auto &tree : threadResult.trees) {
```

Comment:
btw: There's one change we need to make here before streaming has user impact.

Right now, the worker threads typecheck files until there are no more, and then send the trees back to the coordinator thread -- so the coordinator thread won't flush those files until the ~end of typechecking. We need to change the loop so that workers periodically send back what they have.